app/models/document_factory_registry.rb

```diff
@@ -6,7 +6,7 @@
 require "builders/manual_document_builder"
 require "manual_with_documents"
 require "slug_generator"
-require "specialist_document"
+require "section"
```

Comment:
Possibly not - I think we'll take a look at this after we merge this PR, there's probably a few unnecessary requires scattered around.